.travis.yml

```diff
@@ -128,13 +128,30 @@ jobs:
       language: python
       python: "3.6"
       script:
+        - sed -i "s/.0$/.$TRAVIS_BUILD_NUMBER/" game-core/src/main/resources/META-INF/triplea/product.properties
+        - version=$(cat game-core/src/main/resources/META-INF/triplea/product.properties | sed 's/.* //')
         - eval "$(ssh-agent -s)"
-        - version=$(sed "s/.0$/.$TRAVIS_BUILD_NUMBER/" game-core/src/main/resources/META-INF/triplea/product.properties)
         - cd infrastructure
-        - export ANSIBLE_HOST_KEY_CHECKING=False
+        - ./build_artifacts
         - echo "$ANSIBLE_VAULT_PASSWORD" > vault_password
         - ./run_deployment "$version" -i ansible/inventory/prerelease
-
+    - stage: deploy production
+      if: (branch = master) and (repo = 'triplea-game/triplea') and (type != 'pull_request')
+      addons:
+        apt:
+          packages:
+            - sshpass
+      install:
+        - pip install ansible
+      language: python
+      python: "3.6"
+      script:
+        - version="2.0.16817"
```

Comment:
Is this supposed to be hardcoded?
It doesn't seem convenient having to update this file whenever we want to push a new version, so I'm guessing you just put that there to verify this step on your fork

Reply:
No, it is hardcoded. We won't push a new version too often.
 
Travis env variable was another option, but we lose the record of when updates are made.

It's perhaps important to note this is just the version of the downloaded artifacts. We will run the rest of the deployment configuration on every update. This will allow us to deploy/remove keys and do security or configuration updates without changing the running binaries.

Triggering builds on branches or tags was also another option but also not great either as we can't update prod with the latest non-source code updates.

Reply:
@ron-murhammer heads up that updating this version value is how we would change the version of the running software in prod.

@RoiEXLab , I'd recommend we extract this conversation to an issue so we can be sure to be on the same page, and/or consider if there are other ways to inject and maintain the version we wish to deploy.

Reply:
For follow-up to this topic: https://github.com/triplea-game/triplea/issues/5602
I'd recommend we avoid blocking this PR as we are still landing new features and can have room to iterate. It's also the case that we are not yet updating production in any way, so a new PR could cleanly show the changes, rather than waiting, losing context, and then having to re-review everything to regain the context (and avoids further merge conflicts)